addons/account_edi/models/account_edi_document.py

```diff
@@ -170,10 +170,20 @@ def _postprocess_post_edi_results(documents, edi_result):
                     if not old_attachment.res_model or not old_attachment.res_id:
                         attachments_to_unlink |= old_attachment
                 else:
+                    blocking_level = move_result.get('blocking_level', DEFAULT_BLOCKING_LEVEL) if move_result.get('error') else False
                     document.write({
                         'error': move_result.get('error', False),
-                        'blocking_level': move_result.get('blocking_level', DEFAULT_BLOCKING_LEVEL) if 'error' in move_result else False,
+                        'blocking_level': blocking_level,
                     })
+                    title = _('An error occurred during posting electronic document')
+                    document.move_id.message_post(
+                        title=title,
+                        body=move_result.get('error', False),
+                    )
+                    # to be sure blocking_level is one of the standard values and otherwise switch to the default one
+                    blocking_level = (blocking_level or '').upper()
+                    log_level = logging._nameToLevel.get(blocking_level, logging._nameToLevel[DEFAULT_BLOCKING_LEVEL.upper()])
+                    _logger.log(log_level, '%s: %s', title, move_result.get('error', False))
```

Comment:
Although it might be useful for proactively monitoring such errors, I fear that this can generate a lot of useless alerts. I don't think it is really necessary to add this part.